### PR TITLE
chore: Removed Python 2.7 support

### PR DESCRIPTION
**Pull request by rennerocha**
…ed code.

### CODE REVIEW

**Comment by Gallaecio** [comment]
Since the documentation says 3.5+, should this patch also include official support for 3.8?

**Comment by rennerocha** [comment]
Build is passing:
https://travis-ci.org/github/scrapinghub/spidermon/builds/690104886

**Comment by wRAR** [comment]
`six` is still listed at least in setup.py and requirements.txt. Or is it still used by some code?

**Comment by rennerocha** [comment]
> `six` is still listed at least in setup.py and requirements.txt. Or is it still used by some code?

`six` is still being used in several places. I only removed parts of code that are explicitly related to Python 2 (places where we find `if six.PY2` for example). I was a bit conservative to avoid removing more code than needed.

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,6 @@ git:
 language: python
 dist: xenial
 python:
-  - 2.7
   - 3.5
   - 3.6
   - 3.7
```
Comment (Gallaecio) [inline .travis.yml:10]:
```suggestion
  - 3.7
  - 3.8
```

**Comment by rennerocha** [comment]
Build is passing:
https://travis-ci.org/github/scrapinghub/spidermon

I removed Python 3.8 as it was breaking the build and it will need a better investigation how to fix it.
This PR will only contain changes necessary to drop Python 2.7 support. A new issue will be created to add Python 3.8 support.

**Pull request merged**